CustomerContactBook/Controllers/CustomerGroupsController.cs

```diff
@@ -57,25 +55,16 @@ public async Task<ActionResult<CustomerGroup>> GetCustomerGroup(int id)
         /// <param name="id">id of group</param>
         /// <param name="customerGroup">customerGroup to add</param>
         [HttpPut("{id}")]
-        public async Task<IActionResult> PutCustomerGroup(int id, CustomerGroup customerGroup)
+        public async Task<IActionResult> PutCustomerGroup(int id, CustomerGroupModel customerGroup)
         {
             if (id != customerGroup.Id)
```

Comment:
I don't recommend having `id` in the update model - this parameter should be only in the query path.